openedx/core/djangoapps/schedules/management/commands/tests/test_send_course_update.py

```diff
@@ -35,3 +42,13 @@ def setUp(self):
         mock_highlights = patcher.start()
         mock_highlights.return_value = ['Highlight {}'.format(num + 1) for num in range(3)]
         self.addCleanup(patcher.stop)
+
+    @ddt.data(
+        ExperienceTest(experience=ScheduleExperience.EXPERIENCES.default, offset=expected_offsets[0], email_sent=False),
+        ExperienceTest(experience=ScheduleExperience.EXPERIENCES.course_updates, offset=expected_offsets[0], email_sent=True),
+    )
```

Comment:
I couldn't figure out how to get `ddt` to pull its test data from the sub-class when inheriting from the base test class. Does anyone on @edx/rapid-experiments-team know how to do this?